src/main/java/com/hubspot/jinjava/objects/collections/PyMap.java

```diff
@@ -58,4 +58,15 @@ public void putAll(Map<? extends String, ? extends Object> m) {
     }
     super.putAll(m);
   }
+
+  @Override
+  public int hashCode() {
```

Comment:
what's the default implementation of this look like? You could create an implementation that kept a set of all the hashcodes.

Reply:
default implementation ```
public int hashCode() {
        int h = 0;
        for (Entry<K, V> entry : entrySet())
            h += entry.hashCode();
        return h;
    }
```